src/Chip/Chip.js

```diff
@@ -0,0 +1,168 @@
+// @flow weak
+
+import React, { PropTypes } from 'react';
+import { createStyleSheet } from 'jss-theme-reactor';
+import classNames from 'classnames';
+import DeleteIcon from '../svg-icons/navigation/cancel';
+
+export const styleSheet = createStyleSheet('Chip', (theme) => {
+  const { palette, shadows, transitions } = theme;
+  return {
+    root: {
+      fontSize: 13,
+      display: 'flex',
+      alignItems: 'center',
+      justifyContent: 'center',
+      height: 32,
+      outline: 'none',
+      border: 0,
+      color: palette.getContrastText(palette.grey[300]),
+      backgroundColor: palette.grey[300],
+      boxShadow: shadows[0],
+      borderRadius: 16,
+      whiteSpace: 'nowrap',
+      width: 'fit-content',
+      transition: transitions.create(),
+    },
+    clickable: {
+      '&:active': {
+        boxShadow: shadows[1],
+      },
+      '&:hover, &:active': {
+        backgroundColor: palette.grey.A100,
+      },
+      cursor: 'pointer',
+    },
+    avatar: {
+      marginRight: -4,
+      width: 32, // Added for jss avatar
+      height: 32, // Added for jss avatar
+      fontSize: 16,
+    },
+    avatarIcon: {
+      width: 19.2, // Added for jss avatar
+      height: 19.2, // Added for jss avatar
```

Comment:
Not sure - in `master` this is a dynamic style in Avatar, where the icon is 0.6 * avatar `size`. Since Avatar no longer supports this, we have to calculate it manually (32 * 0.6 = 19.2). 

Happy to round it down if you think it could cause a problem.